server/src/main/java/org/elasticsearch/index/translog/Translog.java

```diff
@@ -1010,16 +1009,17 @@ public Source(BytesReference source, String routing) {
     public static class Index implements Operation {
 
         public static final int FORMAT_6_0 = 8; // since 6.0.0
-        public static final int FORMAT_NO_PARENT = FORMAT_6_0 + 1; // since 7.0
-        public static final int SERIALIZATION_FORMAT = FORMAT_NO_PARENT;
+        public static final int FORMAT_7_0 = FORMAT_6_0 + 5; // reserve some versions for 6.x
```

Comment:
I don't think we should preemptively do this. If it comes to us requiring the extra versions for a change in 6.x, and we have not released 7.0, we can up the format version for the current change. If we have released 7.0 at that point, we cannot make any changes to the format anyhow.

Reply:
+1

Reply:
+1